.github/workflows/pre-submit.yaml

```diff
@@ -13,17 +13,20 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
     - name: Checkout code
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
       with:
         fetch-depth: 0
 
     - name: Set up Go
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v3
       with:
         go-version: 1.18
 
-    - name: Build and test
-      run: make manifests generate docker-build
+    - name: Run checks and unit tests
+      run: make test
```

Comment:
IIUC test is redundant here.
It also runs as part of _docker-build_ which is run as part of _container-build_

Reply:
In NHC and NMO we run `test` twice -  as a step before we try to build the containers and with _container-build_ , but with `test-no-verify`.
Why have we done that? IIUC since in container-build we have been using test-no-verify we wanted to run `test` beforehand while in SNR it is not the case as it doesn't have test-no-verify.
Therefore, it is indeed redundant